ibis/backends/base/sql/__init__.py

```diff
@@ -20,6 +21,11 @@
 ]
 
 
+def _find_memtables(expr):
+    is_in_memory_table = isinstance(op := expr.op(), ops.InMemoryTable)
+    return lin.proceed, op if is_in_memory_table else None
```

Comment:
Nit, 100% feel free to ignore.

I find the usage of the walrus operator in locations other than `if`/`while` loops kinda hard to read. I'm not used to seeing variables defined inside a call to a function, my brain doesn't expect to look for them there.

```suggestion
    op = expr.op()
    return lin.proceed, op if isinstance(op, ops.InMemoryTable) else None
```

Reply:
That's fair. I don't feel strongly about this stylistic choice here.